8.x/getting-started/installation.md

```diff
@@ -269,152 +557,322 @@ Run the following command to install Composer in the /usr/local/bin directory:
 
 ```bash
 cd public_html
 cp .htaccess.example .htacces
 ```
 
-## Setup the project template
+## Setup project template
+
+- Fork and clone the `larammerce_base_theme` project from [larammerce github](https://github.com/larammerce/larammerce-base-theme).
 
-According to the Larammerce project structure, the project template and backend are developed dedicated and independent of each other.
 
-Clone the template project:
+- Install `direnv` tool.
+  
+  `direnv` is necessory for development to create different environment in each directory:
 
+ See how to install direnv [here](https://direnv.net) or follow this instruction:
+
+ On `larammerce_base_theme` directory run:
 ```bash
-cd /var/www/larammerce/data
-clone https://github.com/path/to/template-project.git
+cd path/to/larammerce_base_theme
+sudo apt install direnv
 ```
-Install npm dependencies after cloning the project
+To hook direnv into your shell, add the following line to the end of `~ /.bashrc` file by these steps:
+1. Open the `/.bashrc` file.
 
 ```bash
-cd template-project
-npm install
+cd path/to/larammerce_base_theme
+vim ~/.bashrc
 ```
 
-Then to build the project resource files run:
+2. Enable editing with `shift+g+o+enter`.
+3. Add the line below at the end of the file:
 ```bash
-npm run prod # to build and minify the project resource files.
-npm run watch # to watch for resource file changes and then build and export them after every change.
+eval "$(direnv hook bash)"
+``` 
+- Open `.envrc` and add this line on top of the file:
+```bash
+export ECOMMERCE_BASE_PATH=/home/your-sysytem-name/your-root/larammerce
 ```
+Note: On the main directory run `pwd` to get the path used in the command above.
 
-Create a new .envrc file from the .envrc.example file
-
+Here, you may run to an access issue, to solve it run:
 ```bash
-cp .envrc.example .envrc
+direnv allow . 
 ```
-Add the following to the file named `.envrc`:
 
-```bash
-export ECOMMERCE_BASE_PATH=/var/www/larammerce 
+To test the accuracy of this command, run:
+
+``` bash
+echo  $ECOMMERCE_BASE_PATH
 ```
+The result should be the path you equalized before.
+
+Run the following command to install node requirements:
 
-Then this command to make the direnv package know this directory as its list of directories:
+```bash
+cd path/to/the/larammerce/project
+npm install
+```
+Then run this command to build the resource files:
 
 ```bash
-direnv allow .
+npm run prod 
 ```
-And after each change in the template, we enter this command again to deploy the resources to the backend directory:
+
+
+ Everything is ready to deploy the project on `localhost:8080`. Execute:
+ ```bash
+ php -S 0.0.0.0:8080 -t public_html/
+ ```
+
+ To enable a basic theme on the project, run this command on ` ~/.../larammerce-base-theme` :
 
 ```bash
 ./deploy.sh
 ```
 
+---
+
+#### Working with the theme
+
+As a sample, lets build a web page! 
+
```

Comment:
In the browser, type "http://0.0.0.0:8080/admin" to see the login page.
In the terminal, go to the `~/projects/larammerce/database/seeders/` directory and run:
```bash
vim UsersTableSeeder.php
```
In this file, you can see the username and password for admin user. Insert these values into the login fields to enter the admin panel.